packages/sanity/src/structure/panes/document/comments/CommentsWrapper.tsx

```diff
@@ -60,9 +58,9 @@ function CommentsProviderWrapper(props: CommentsWrapperProps) {
         inspect: COMMENTS_INSPECTOR_NAME,
         comment: commentId,
       })
-      return `${studioUrl}${intentLink}`
+      return `${window.location.origin}${intentLink}`
```

Comment:
Likely silly question but worth asking, in other places we have guardrails to check if the window exists, could this potentially break when the window location is not available or is this something we don't need to be concerned about?